src/System.IO.Packaging/src/System/IO/Packaging/ContentType.cs

```diff
@@ -662,14 +662,8 @@ private static bool IsLinearWhiteSpaceChar(char ch)
             {
                 return false;
             }
-
-            foreach (char c in s_linearWhiteSpaceChars)
-            {
-                if (ch == c)
-                    return true;
-            }
-
-            return false;
+            var whiteSpaceIndex = Array.IndexOf(s_linearWhiteSpaceChars, ch);
```

Comment:
Nit: could you put back the blank line above this one?  We like to have blank lines after the end of blocks like this.